src/main/java/org/folio/circulation/domain/StoredRequestRepresentation.java

```diff
@@ -22,6 +23,7 @@ public JsonObject storedRequest(Request request) {
     addStoredInstanceProperties(representation, request.getInstance());
     addStoredRequesterProperties(representation, request.getRequester());
     addStoredProxyProperties(representation, request.getProxy());
+    addSearchIndexProperties(representation, request, request.getItem());
```

Comment:
Can we get rid of 3rd parameter and extract it from within `addSearchIndexProperties`?
```suggestion
    addSearchIndexProperties(representation, request);
```

Reply:
sure, thanks